PowerEditor/src/Parameters.cpp

```diff
@@ -4020,6 +4020,31 @@ void NppParameters::feedGUIParameters(TiXmlNode *node)
 			}
 		}
 
+		else if (!lstrcmp(nm, TEXT("SmartHighLightMode")))
+		{
+			TiXmlNode *n = childNode->FirstChild();
+
+			if (n)
+			{
+				const TCHAR* val = n->Value();
+				if (val)
+				{
+					if (!lstrcmp(val, TEXT("1")))

```

Comment:
True, we could probably do a static_cast; I personally don't think it makes it any more readable or more performant, though.